build/ooceapps/build.sh

```diff
@@ -54,7 +54,7 @@ add_extra_files() {
 }
 
 init
-download_source "v$VER" $PROG $VER
+download_source "v$VER" $PROG $PROG
```

Comment:
this should not be changed.

Reply:
https://github.com/omniosorg/omnios-extra/pull/47/files#diff-2df8302a6d21884fc141f74199cfce28R31 << but version should be changed here to `VER=0.2.0`